.github/workflows/ci.yml

```diff
@@ -6,50 +6,80 @@ on:
 
 env:
   BUILD_TYPE: Release
+  WORKING_DIRECTORY: ${{ github.workspace }}/build
 
 jobs:
   build-mingw:
     runs-on: ubuntu-24.04
 
     steps:
-
     - name: Install packages
       run: |
         sudo dpkg --add-architecture i386
         sudo apt-get update
         sudo apt-get install --no-install-recommends g++-mingw-w64-i686-posix ninja-build wine32 wine
+      shell: bash
```

Comment:
why set `shell` everywhere? On Linux it is Bash by default

Reply:
Agreed, I'll remove it from `build-mingw`.

What about Windows? Do I keep it as an explicit value, as `Git` provides a `Bash` shell within Windows-based runners?